library/src/main/res/layout/exomedia_video_view_layout.xml

```diff
@@ -17,4 +17,9 @@
         android:layout_height="match_parent"
         android:scaleType="centerCrop"
         tools:ignore="ContentDescription"/>
+
+    <com.google.android.exoplayer2.ui.SubtitleView
```

Comment:
I've purposefully not included the UI package, we should handle this on our own instead of depending on the one they provide (that is specifically created for their simple exo player)